packages/flutter_tools/lib/src/linux/linux_device.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../application_package.dart';
+import '../base/platform.dart';
+import '../build_info.dart';
+import '../desktop.dart';
+import '../device.dart';
+import 'linux_workflow.dart';
+
+/// A device which represents a desktop linux target.
```

Comment:
Grammar nit: s/which/that/

(https://www.grammarly.com/blog/which-vs-that/)

... same comment in the other device.dart files.